code-comment-style.md

```diff
@@ -0,0 +1,129 @@
+# Code Comment Style

```

Comment:
Maybe this file need to be capitalized like other md files in the project root?

Reply:
Good point. Will update accordingly.